mods/cnc/rules/defaults.yaml

```diff
@@ -480,10 +480,13 @@
 		SellSounds: cashturn.aud
 	Capturable:
 	WithMakeAnimation:
+		MakeUpgrades: underConstruction
+	DisableOnUpgrade:
+		UpgradeTypes: underConstruction
```

Comment:
We have yaml strings which are fully lowercase, even for multiple words. I haven't seen any example of camelcase usage in the repository, but I haven't been reading it much lately. I'd rather we didn't introduce yet another capitalisation scheme to the yaml.